client/assets/html/index.html

```diff
@@ -8,11 +8,10 @@
     <meta name="description" content="Timesheet Demo Application"/>
     <link href="/assets/css/style.css" rel="stylesheet" type="text/css"/>
   </head>
-  <body ng-controller="MainCtrl">
+  <body ng-controller="MainCtrl as mainCtrl">
     <div ng-cloak="ng-cloak" class="tsz-page-container">
-        <div ui-view="navbar"></div>
-        <div ui-view="content"></div>
-        <div ui-view="login"></div>
```

Comment:
It's not being used. But, since this file was generated from index.jade, it wiped out the login include. I went back and looked at the lab and the 'copy paste' chunk includes the login div. To keep things consistent, i'll update the jade file.